opal/runtime/opal_progress.c

```diff
@@ -235,10 +239,14 @@ opal_progress_event_users_increment(void)
 void
 opal_progress_event_users_decrement(void)
 {
+#if OPAL_ENABLE_DEBUG || ! OPAL_PROGRESS_USE_TIMERS
```

Comment:
it looks correct to me, we only need ```val``` in order to print it (e.g. ```OPAL_ENABLE_DEBUG```) or a few lines later if ```! OPAL_PROGRESS_USE_TIMERS```